RELEASE.md

```diff
@@ -15,7 +15,7 @@ Bug fixes are expected to be worked on and applied to `master`.  If the fix is n
 The KUDO Project is released on an as-needed basis. The process is as follows:
 
 1. An issue is proposing a new release with a changelog since the last release
```

Comment:
let's ignore for now that these are not actually our release practices right now and tackle it in different PR